src/pages/home/ReportScreen.js

```diff
@@ -24,16 +30,26 @@ const propTypes = {
 
     /** Tells us if the sidebar has rendered */
     isSidebarLoaded: PropTypes.bool,
+
+    /** Whether or not to show the Compose Input */
+    session: PropTypes.shape({
+        shouldShowComposeInput: PropTypes.bool,
```

Comment:
NAB: Why is this nested in `session` and not its own prop?

Reply:
I can't remember if I did this or if someone else did. But I'm gonna guess it just seemed more convenient than creating a new key to track this.